e2e/tests/oai/004-ran-network.sh

```diff
@@ -78,6 +78,6 @@ k8s_wait_ready_replicas "deployment" "oai-gnb-du" "$_edge_kubeconfig" "oai-ran-d
 # Check if the NGAPSetup Request Response is okay between AMF and CU-CP
 _wait_for_ran "$_regional_kubeconfig" "Received NGAP_REGISTER_GNB_CNF: associated AMF" "N2"
 # Check if the E1Setup Request Response is okay between CU-CP and CU-UP
-_wait_for_ran "$_regional_kubeconfig" "e1ap_send_SETUP_RESPONSE" "E1"
+_wait_for_ran "$_regional_kubeconfig" "Accepting new CU-UP ID" "E1"
```

Comment:
What is the reason for changing this? Do we need to make these changes going foward as well?

Reply:
@vjayaramrh Because we updated the version of OAI-RAN we are using. R2 was using a really old version and if we keep using that we can't do bandwidth reconfiguration which was one of the Service assurance use cases. So even in the future I will keep upgrading OAI version.

Reply:
@arora-sagar Thanks for explaining.
Suggestion(if it makes sense):would treating it as a exported variable in a release specific file and sourcing it help instead of changing this file directly.?

Reply:
Yeah, that would be a good solution.